content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -1004,13 +1004,10 @@ observe that pods from a Job are stuck with the tracking finalizer.
 
 ### Elastic Indexed Jobs
 
-{{< feature-state for_k8s_version="v1.27" state="beta" >}}
+{{< feature-state feature_gate_name="ElasticIndexedJob" >}}
 
 You can scale Indexed Jobs up or down by mutating both `.spec.parallelism` 
-and `.spec.completions` together such that `.spec.parallelism == .spec.completions`. 
-When the `ElasticIndexedJob`[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
-on the [API server](/docs/reference/command-line-tools-reference/kube-apiserver/)
-is disabled, `.spec.completions` is immutable.
+and `.spec.completions` together such that `.spec.parallelism == .spec.completions`.
```

Comment:
Worth adding that, when scaling down, Kubernetes removes the pods with the higher indexes.

Reply:
thanks, added